test/components/notificationBarTest.js

```diff
@@ -116,7 +116,7 @@ describe('notificationBar', function () {
       }).click('button=Yes')
       .tabByIndex(0)
       .loadUrl('about:passwords')
-      .waitForExist('tr.passwordItem')
+      .waitForExist('.passwordItem')
```

Comment:
it should be `.waitForExist('[data-test-id="passwordItem"]')`

You can check if it works by running `npm run watch-test` and on other terminal tab run
```
npm run test -- --grep="autofills remembered password on login form"
```